<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="main_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Arkiv</string>
    </property>
    <addaction name="new_action"/>
    <addaction name="separator"/>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="save_as_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="input_menu">
    <property name="title">
     <string>Indata</string>
    </property>
    <addaction name="segments_action"/>
    <addaction name="bc_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="input_menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new_action"/>
   <addaction name="open_action"/>
   <addaction name="save_action"/>
   <addaction name="save_as_action"/>
   <addaction name="separator"/>
   <addaction name="segments_action"/>
   <addaction name="bc_action"/>
  </widget>
  <action name="open_action">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar/images/icons8-open-32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Öppna</string>
   </property>
  </action>
  <action name="save_action">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar/images/icons8-save-32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Spara</string>
   </property>
  </action>
  <action name="save_as_action">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar/images/icons8-save-as-32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Spara som...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Avsluta</string>
   </property>
  </action>
  <action name="new_action">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar/images/icons8-file-32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ny modell</string>
   </property>
  </action>
  <action name="segments_action">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar/images/icons8-edit-property-32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Balkar...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="bc_action">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar/images/icons8-edit-property-32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Stöd...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="beam_app.qrc"/>
 </resources>
 <connections/>
</ui>
